server/migrations/20200405204013-create-store-updates.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('StoreUpdates', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      store_id: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: {
+            tableName: 'StoreInfos'
+          },
+          key: 'id'
+        },
+      },
+      ip: {
+        type: Sequelize.STRING
+      },
+      user_id: {
```

Comment:
the user_id passed above was -1. should this be an integer field?

Reply:
Cast to string is happening automatically